web/src/constants/dimensions.ts

```diff
@@ -30,6 +31,7 @@ const dimensions: DimensionsType = {
   footerHeight: 50,
   navigationMenuHeight: 90,
   poiDesktopPanelWidth: 332,
+  additionalSpacing: 35,
```

Comment:
In my opinion we should not put this in this dimensions.ts and instead just use it directly as a magic number. Reasons:
* It is only used in one place and probably really specific to this one use case
* The naming is really generic and might mean anything and probably leads to more confusion than it helps

Personally I would also prefer some power of two number like `32px` but thats just a personal opinion.